cirq/ops/pauli_string_test.py

```diff
@@ -48,6 +48,38 @@ def test_eq_ne_hash():
         eq.add_equality_group(cirq.PauliString({q: p0, q2: p1}, False))
 
 
+def test_equal_up_to_sign():
```

Comment:
Hmmm. Maybe we should add an `equater` lambda to `EqualsTester`. That would massively simplify this test.

Reply:
Maybe if there are more cases like this.